test/Microsoft.OpenAPI.OData.Reader.Tests/Resources/Basic.OpenApi.json

```diff
@@ -293,6 +293,27 @@
         "x-ms-docs-operation-type": "operation"
       }
     },
+    "/City/$count": {
```

Comment:
City looks like a singleton. 
It's not allowed to have $count after a single navigation property or singleton.

Reply:
city is actually also an entity set, arguably it should be names cities but I didn't want to impact the source file.
https://github.com/microsoft/OpenAPI.NET.OData/blob/13427946decd6bbe1ba5e7b52604d32698f455d1/test/Microsoft.OpenAPI.OData.Reader.Tests/Common/EdmModelHelper.cs#L273

Reply:
Oh, ok. My error